stubs/docker/docker/models/containers.pyi

```diff
@@ -79,7 +79,7 @@ class Container(Model):
     def restart(self, **kwargs): ...
     def start(self, **kwargs) -> None: ...
     def stats(self, **kwargs): ...
-    def stop(self, **kwargs) -> None: ...
+    def stop(self, *, timeout: float | None = None) -> None: ...
```

Comment:
I chose `float` rather than `int` (`int` is documented) because of:

               I think it also accepts floats for the timeout. The timeout gets passed to `requests.Session.post()`.

_Originally posted by @Akuli in https://github.com/python/typeshed/pull/12037#discussion_r1615132603_